server/index.js

```diff
@@ -52,9 +52,14 @@ app.get('/api/full/photos/:propertyId', (req, res) => {
 // BEGIN SDC ROUTES
 //==================
 
-var { getRoute, postRoute, putRoute, deleteRoute} = require('./controller');
+var { postRoute, putRoute, deleteRoute } = require('./controller');
+var parser = require('body-parser');
 
-app.get('/api/')
+app.post('/api/basicdetails/', parser.json(), postRoute);
+
+app.put('/api/basicdetails/:propertyId', parser.json(), putRoute);
+
+app.delete('/api/basicdetails/:propertyId', deleteRoute);
```

Comment:
you might want to refactor these to conform to REST, i.e. something like `/properties/:propertyId/photos` -> this route should be descriptive of what information you are addressing, so, you're going through the route of properties, selecting a specific property, then getting all photos from that property. I know that you have legacy code to work with so it's up to you whether you want to address this in the future or not

Reply:
@elizabethcan will address this after writing seed generation script